app/api/dao/user.py

```diff
@@ -311,6 +311,9 @@ def confirm_registration(token):
         else:
             user.is_email_verified = True
             user.email_verification_date = datetime.now()
+            # gravatar
+            gravatar = Gravatar(user.email)
+            user.photo_url = gravatar.get_image(default = "identicon")
```

Comment:
Hey, we need to add a gravatar only if `user.photo_url` is empty. We need to check that first.

Reply:
Done please review :)